lib/index.js

```diff
@@ -36,6 +37,29 @@ class Client {
     throw 'invalid URL argument'
   }
 
+  /**
+   * Make action request CKAN. For more info check
+   * https://docs.ckan.org/en/latest/api/index.html#action-api-reference
+   * @param {string} actionName - The action name, e.g. site_read, package_show ...
+   * @param {object} payload - The payload being sent to CKAN
+   */
+  async action(actionName, payload) {
```

Comment:
What about option to force using http get - see https://github.com/datopian/ckan-client-js/issues/10

This is secondary so if we don't have it for now that's ok - we should just open an issue so we remember to do it if we close #10